hack/run-e2e-kind.sh

```diff
@@ -30,17 +30,28 @@ export CLUSTER_CONTEXT="--name ${CLUSTER_NAME}"
 
 export KIND_OPT=${KIND_OPT:="--config ${VK_ROOT}/hack/e2e-kind-config.yaml"}
 
-function get-k8s-server-version {
-    echo $(kubectl version --short=true | grep Server | sed "s/.*: v//" | tr "." " ")
+function check-jq {
```

Comment:
What is the relationship between jq version and k8s version?

Reply:
Sorry, I don't quite understand what you mean.
1. we get server version information of cluster by kubectl version --short=true in function get-k8s-server-version now.
2. but --short flag of kubectl has be deprecated.So we get an error as issue #3476 shown when we run e2e test.
3.  then I think that we should get the version information of server by another way.
4. jq is a json parser in shell environments. And it is written in C.
5. So the version of jq is not related to the version of kubernetes.